src/renderer/components/home.vue

```diff
@@ -1,37 +1,43 @@
 <template>
-    <div class="grid-container fluid text-center">
-        <div class="grid-x grid-padding-y">
-          <div id="challenge" class="cell small-2"></div>
-          <div class="cell small-8">
-              <h2>Tournament Management System</h2>
-              <h4>v2018.2.0</h4>
-          </div>
-          <div id="fll" class="cell small-2"></div>
+    <div class="ui center aligned grid container">
+        <div class="row">
+            <div id="challenge" class="three wide column"></div>
+            <div class="ten wide column">
+                <h2>Tournament Management System</h2>
+                <h4>v2019.0.0</h4>
+            </div>
+            <div id="fll" class="three wide column"></div>
         </div>
-        <div class="grid-y">
-          <div class="cell grid-x grid-margin-x">
-            <fieldset v-for="module in modules" class="cell small-3 fieldset">
-              <legend class="text-left">
-                <a @click="event => openSite(event, module.site)" :href="module.site">{{module.name}}</a>
-              </legend>
-              <label>
-                <div><qrcode :value="module.site"></qrcode></div>
-                <a @click="event => openSite(event, module.site)" :href="module.site">{{module.site}}</a>
-                <button class="button" @click="() => saveInClipboard(module.site)"><i class="fas fa-copy"></i>&nbsp;Copy link</button>
-              </label>
-            </fieldset>
-          </div>
+        <div class="row ui four cards">
+            <div v-for="module in modules" class="card">
+                <div class="content">
+                    <a class="header" @click="event => openSite(event, module.site)" :href="module.site">
+                        {{module.name}}
+                    </a>
+                    <div class="ui divider"></div>
+                    <div class="description">
+                        <qrcode :value="module.site"></qrcode>
+                    </div>
+                </div>
+                <div class="extra content">
+                    <a class="header" @click="event => openSite(event, module.site)" :href="module.site">
+                        {{module.site}}
+                    </a>
+                    <div class="ui button" @click="() => saveInClipboard(module.site)">
```

Comment:
There is very little space here. Use an `primary link icon` and remove the text. The icon is clear enough and it's redundent here. Also make the icon float right.

Reply:
I don't really think it's a clear thing, and we also have enough space. @alanggreen what do you think?

Reply:
![image](https://user-images.githubusercontent.com/19326938/61583146-ad932580-ab3c-11e9-80ce-34c132d7408a.png)
Just how it would look with a button... I think it's clear

Reply:
We as a developer tends to know this one because it's very useful in website we works with. But it's not very common outside of programming website.
Also we don't really know what this icon means without it's context since if you will check in different developing tools they are using different icons.

Reply:
In most cases you're right, and icons should not be displayed without text, but the few common actions that include clipboard, file management, text editing and probably a few more that I forgot, when the context is clear it's OK to leave them as icons.
Specifically text copying buttons, if the text is single lined they should be at the same line as the text, otherwise the context is unclear. This is why I wanted an icon here, which will make sure it's beside the text and not under it. If you can make it work with a text, that doesn't take the focus from the link then go ahead.
But generally you're right and therefore I'll add hover texts to several icons in the scoring cards, and chagne the standard so that links at the top of the page will have texts.

BTW this is a very common icon...
Just an example (picture below).
![image](https://user-images.githubusercontent.com/19326938/61594430-4fc21480-abf4-11e9-989c-5540e1cadeb5.png)